src/main/resources/db/migration/core/V1.08__CLIENT_USER.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE client_user
+(
+    id VARCHAR(50) NOT NULL PRIMARY KEY,
+    client_id    VARCHAR(50) NOT NULL,
+    address_id  VARCHAR(50) NOT NULL,
```

Comment:
this should link with user_id table.

Reply:
this is many to many table, its client to user table

Reply:
+ user_id = username, at least as I understood